src/commands/member_services/print.js

```diff
@@ -0,0 +1,83 @@
+const Command = require('../Command');
+const { getPdfNumOfPages } = require('../util/getPdfNumOfPages');
+const {
+  validateDiscordID,
+  printerHealthCheck,
+  pushDiscordPDFToSqs,
+  editUserPagesPrinted
+} = require('../../APIFunctions/Print.js');
+
+module.exports = new Command({
+  name: 'print',
+  description: 'Print PDF file',
+  aliases: ['print'],
+  example: 's!print',
+  permissions: 'member',
+  category: 'member services',
+
+  // eslint-disable-next-line no-unused-vars
+  execute: async (message, args) => {
```

Comment:
do we need this line?
```suggestion
  execute: async (message) => {
```